src/index.scss

```diff
@@ -198,13 +203,17 @@ html, body, #root, #app, .full-height {
   }
 }
 
+.hero-img.mobile-hide { //temporary, push up the desktop hero image a bit
```

Comment:
Temporary?

Reply:
Styling will need to be moved when we refactor to use only one hero image w/ shifting layout instead of two hero images